benchmarks/test_bwd_bn/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Include the DNNMark include path
+include_directories(${DNNMARK_INCLUDES})
+
+# Obtain the name of the directory(also part of the benchmark name)
+get_filename_component(BENCHMARK_DIR ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+
+# Set the benchmark name
+set(BENCHMARK_NAME ${PROJECT_NAME}_${BENCHMARK_DIR})
+
+# Add source file
+file(GLOB_RECURSE BENCHMARK_SRC_FILE RELATIVE 
+     ${CMAKE_CURRENT_SOURCE_DIR} 
+     *.cc)
+
+# Print out the source file information
+message(STATUS ${BENCHMARK_NAME} ": Benchmark source files: " ${BENCHMARK_SRC_FILE})
+
+# Add benchmark executable
+add_executable(${BENCHMARK_NAME} ${BENCHMARK_SRC_FILE})
```

Comment:
Right now the source of each benchmark has been modified just a little bit. Could you modify that as well? Thanks a lot!